acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/lotus/AddressesUtil.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.eth2signer.dsl.lotus;
+
+import static tech.pegasys.eth2signer.dsl.lotus.FilecoinKeyType.BLS;
+import static tech.pegasys.eth2signer.dsl.lotus.FilecoinKeyType.SECP256K1;
+
+import java.util.Map;
+
+public class AddressesUtil {
```

Comment:
can we make the keys/address dynamic - i.e. during tests, randomly create private keys, then import them to Filecoin as well as write them to a raw yaml file for Eth2Signer?

Reply:
done. Addresses are generated at runtime via Lotus and set up to be loaded by Signer instance.